maintenance.md

```diff
@@ -852,6 +853,14 @@ Hubble image is no longer built by the upstream. If failing to build the image,
 4. Update `MEOWS_VERSION` in `Dockerfile`.
 5. Update `BRANCH` and `TAG` files.
 
+## neco-server-exporter
+
+![Regular Update](./regular_update.svg)
+
+1. Upgrade direct dependencies listed in `go.mod`. Use `go get` or your editor's function.
+2. Update cilium version in `go.mod` to the version used by neco.
+3. Update `TAG` by incrementing the patch revision, e.g. 1.0.1, 1.0.2, ...
```

Comment:
I think we should check base images in the `Dockerfile` and tools in `e2e/Makefile`

Reply:
In this PR, I think just this one is enough.